src/table/__integ__/table-interaction-metrics.test.ts

```diff
@@ -60,7 +60,7 @@ test(
       })
     );
     expect(metrics[0].interactionTime).toBeGreaterThanOrEqual(500);
-    expect(metrics[0].interactionTime).toBeLessThanOrEqual(2000);
+    expect(metrics[0].interactionTime).toBeLessThanOrEqual(5000);
```

Comment:
Nit: it it is the same value and is expected to stay the same value for all the assertions why not use a const to compare against? This way any changes to the value can be done with one line change.

Reply:
Ideally we won't need to update the value again. If the tests exceed the 5000 in the future, we will probably need to refactor the test anyway.